drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/src/test/java/org/drools/workbench/services/verifier/plugin/client/cache/util/DTableUpdateManagerAttributesTest.java

```diff
@@ -0,0 +1,105 @@
+import org.drools.verifier.core.main.Analyzer;
+import org.drools.workbench.models.guided.dtable.shared.model.GuidedDecisionTable52;
+import org.drools.workbench.services.verifier.plugin.client.Coordinate;
+import org.drools.workbench.services.verifier.plugin.client.DTableUpdateManager;
+import org.drools.workbench.services.verifier.plugin.client.DataBuilderProvider;
+import org.drools.workbench.services.verifier.plugin.client.UpdateException;
+import org.drools.workbench.services.verifier.plugin.client.testutil.AnalyzerProvider;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+
+import static org.mockito.Matchers.anySet;
+import static org.mockito.Mockito.verify;
+
+@RunWith(GwtMockitoTestRunner.class)
+public class DTableUpdateManagerAttributesTest {
+
+    private DTableUpdateManager updateManager;
+    private GuidedDecisionTable52 table52;
+    private AnalyzerProvider analyzerProvider;
+
+    @Mock
+    private Analyzer analyzer;
+
+    @Before
+    public void setUp()
+            throws Exception {
+        analyzerProvider = new AnalyzerProvider();
+
+        table52 = analyzerProvider.makeAnalyser()
+                .withAttributeColumn("date-effective")
+                .withAttributeColumn("date-expires")
+                .withPersonAgeColumn("==")
+                .withPersonApprovedActionSetField()
+                .withData(DataBuilderProvider
+                                  .row(new Date(10), new Date(100), 1, true)
+                                  .row(new Date(10), new Date(100), 1, true)
+                                  .end())
+                .buildTable();
+
+        updateManager = analyzerProvider.getUpdateManager(table52,
+                                                          analyzer);
+    }
+
+    @Test
+    public void testSetDateEffectiveToNull() throws
+            Exception,
+            UpdateException {
+        ArrayList<Coordinate> coordinates = new ArrayList<>();
+        Coordinate coordinate = new Coordinate(0,
+                                               2);
+        coordinates.add(coordinate);
+        table52.getData()
+                .get(0)
+                .get(2)
+                .setDateValue(null);
+
+        updateManager.update(table52,
+                             coordinates);
+
+        verify(analyzer).update(anySet());
+    }
+
+    @Test
+    public void testSetDateExpiresToNull() throws
+            Exception,
+            UpdateException {
+        ArrayList<Coordinate> coordinates = new ArrayList<>();
+        Coordinate coordinate = new Coordinate(0,
+                                               3);
+        coordinates.add(coordinate);
+        table52.getData()
+                .get(0)
+                .get(3)
+                .setDateValue(null);
+
+        updateManager.update(table52,
+                             coordinates);
+
+        verify(analyzer).update(anySet());
```

Comment:
I think this test is not very effective if we do not check the content of the `set` passed to the method `update`.